ddtrace/internal/datadog/profiling/stack_v2/CMakeLists.txt

```diff
@@ -0,0 +1,117 @@
+cmake_minimum_required(VERSION 3.19)
+
+# We're building a Python extension and expect to be called from setup.py
+set(EXTENSION_NAME "" CACHE STRING "Name of the extension")
+project(${EXTENSION_NAME})
+
+# Set C++17 as required standard for the whole project.
```

Comment:
is it not strange to set the standard within an extension ? 
Should we not inherit the standard from the project ?
Or set the standard for our target ?
```
https://cmake.org/cmake/help/v3.10/prop_tgt/CXX_STANDARD.html#prop_tgt:CXX_STANDARD
```

Reply:
Indeed, changed this to a more standard way.